src/wallet/coincontrol.h

```diff
@@ -34,6 +34,8 @@ class CCoinControl
     boost::optional<bool> m_signal_bip125_rbf;
     //! Avoid partial use of funds sent to a given address
     bool m_avoid_partial_spends;
+    //! Allows inclusion of dirty (previously used) addresses
+    bool m_allow_dirty_addresses;
```

Comment:
Commit "wallet: add m_allow_dirty_addresses flag to coin control object"

Could initialize here?

Reply:
Everything is initialized in `SetNull()`. In some cases, you can't initialize in the `.h` file because no `gArgs` available at compile time.